### PR TITLE
added descriptions in webhook

### PR DESCRIPTION
**Pull request by FurstLevy**
[issue](https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/issues/111)

### CODE REVIEW

**Comment by unaizorrilla** [comment]
Hi @FurstLevy  
Look good! Can you increment the package version on dependencies.props?

File: src/HealthChecks.UI/Core/Notifications/WebHookFailureNotifier.cs [hunk 1]
```diff
@@ -108,15 +109,23 @@ private async Task SendRequest(string uri, string name, string payloadContent)
...
-                .Where(c => c.Status != UIHealthStatus.Healthy)
-                .Count();
+                .Count(c => c.Status != UIHealthStatus.Healthy);
 
             return $"There is at least {failedChecks} HealthChecks failing.";
         }
 
+        private string GetFailedDescriptionsFromContent(UIHealthReport healthReport)
+        {
+            var failedChecks = healthReport.Entries.Values
```
Comment (unaizorrilla) [inline src/HealthChecks.UI/Core/Notifications/WebHookFailureNotifier.cs]: Probably with Aggregate you can do select and string join in the same sentence

Comment (CarlosLanderas) [reply src/HealthChecks.UI/Core/Notifications/WebHookFailureNotifier.cs]: Line 118 should be "There are"

Comment (FurstLevy) [reply src/HealthChecks.UI/Core/Notifications/WebHookFailureNotifier.cs]: I have never used Aggregate. I hope I did it right.

**Comment by unaizorrilla** [comment]
Merged!

Next time, please don't remove variables from *,csproj, we use dependencies.props to set versions for packages ( and other intersting properties ).

Thanks for contributing

**Pull request merged**